src/api/app/policies/bs_request_policy.rb

```diff
@@ -13,4 +13,11 @@ def handle_request?
     is_author = record.creator == user.login
     record.state.in?([:new, :review, :declined]) && (is_target_maintainer || is_author)
   end
+
+  def can_add_reviews?
```

Comment:
💭  Drop the `can_`  prefix. A policy is always about can and can't....

Reply:
true